### PR TITLE
Decoupling: Move some more dashboard logic to `wp-dashboard`

### PR DESCRIPTION
**Pull request by mohdsayed**
## Context

Miscellaneous dashboard decoupling items from https://github.com/google/web-stories-wp/issues/2918

## To-do

- [x] Remove "See classic WP list view"
- [x] Move left rail navigation settings to `wp-dashboard`
- [x] Move WordPress specific global styles to `wp-dashboard`

## Testing Instructions

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

## Checklist

- [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Partially addresses  https://github.com/google/web-stories-wp/issues/2918

### CODE REVIEW

File: packages/wp-dashboard/src/theme.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { createGlobalStyle } from 'styled-components';
+import { themeHelpers } from '@web-stories-wp/design-system';
+
+export const GlobalStyle = createGlobalStyle`
```
Comment (swissspidy) [inline packages/wp-dashboard/src/theme.js:23]: 👍

File: packages/dashboard/src/components/pageStructure/leftRail.js [hunk 2]
```diff
@@ -238,7 +242,7 @@ function LeftRail() {
       </div>
       <Content>
         <NavList>
```
Comment (swissspidy) [inline packages/dashboard/src/components/pageStructure/leftRail.js]: Should we perhaps not render `NavList` if `leftRailSecondaryNavigation` is an empty array?

Comment (mohdsayed) [reply packages/dashboard/src/components/pageStructure/leftRail.js]: https://github.com/google/web-stories-wp/pull/9380/commits/13b15f4257859a7218ce03a305a646296d0071d2

**Comment by swissspidy** [comment]
There's now a tiny merge conflict here

**Comment by swissspidy** [comment]
Some karma integration tests now need updating

**Review by swissspidy — APPROVED** [review]

**Pull request merged**